README.md

```diff
@@ -76,7 +78,7 @@ In order to run a full node with all the account history you need to remove `par
 
 | Default | Full | Minimal  | ElasticSearch 
 | --- | --- | --- | ---
-| 16G RAM | 120G RAM | 4G RAM | 500G SSD HD, 32G RAM
+| 100G SSD, 16G RAM | 200G SSD, 120G RAM | 80G SSD, 8G RAM | 500G SSD, 32G RAM
```

Comment:
I think these numbers are wrong. SSD is only required for ES, for normal nodes a classic HD is sufficient.

Blockchain size is currently about 33G, don't know why you specify different HD sizes. Logfiles?
Minimal node should work in 4 GB I think (no history).

Reply:
This is recommended sizes.
Even for ES node HD will do the job but slower(there are public mainnet elastic nodes with HD). 
Eventually we will suggest SSD everywhere to make a sync in a reasonable time. Right now i think we can change to HDD Default and Minimal. In minimal we can change back to 4G in RAM.
Disk size of 80G is ok if the chain is 33G, this should be the size of disk someone should purchase to run it, so considering OS and other tools 80G sounds reasonable or at least 60G.

Reply:
We can also add block producer node to the table.

Reply:
@oxarbitrage is right !
We need both API and Witness nodes requirements information IMHO, would be very helpful really !

Reply:
Ok, still for ES SSD is faster and the network is the same. Anyways, added this https://github.com/bitshares/bitshares-core/pull/1380 and also https://github.com/bitshares/bitshares-core/pull/1379 to update the date of the info.

Pretty nice that the pulls to `master` and `develop` will include this commits if we merge them to `release`, no need for opening new pulls.